locale/en.po

```diff
@@ -5859,9 +5854,9 @@ msgid "TUTORIAL_MICROBE_EDITOR_CELL_TEXT"
 msgstr ""
 "This is the cell editor, where you can evolve your species by spending [b]mutation points[/b] (MP) [thrive:icon]MP[/thrive:icon]. Each generation you will always have [b]100 MP[/b] [thrive:icon]MP[/thrive:icon] to spend, so don't bother saving up!\n"
 "\n"
-"The hex in the center of your screen is your [b]cell[/b], which is composed of a single cytoplasm part.\n"
+"The hex in the center of your screen is your species.\n"
 "\n"
-"To continue, select a part from the left panel. Then left-click next to the hex to place it. You can rotate parts with [thrive:input]e_rotate_left[/thrive:input] and [thrive:input]e_rotate_right[/thrive:input]."
+"To continue, scroll down on the left panel until you find the nucleus. Click on it, then left-click next to the hex to place it. You can rotate parts with [thrive:input]e_rotate_left[/thrive:input] and [thrive:input]e_rotate_right[/thrive:input]."
```

Comment:
Yeah, I really think the scrolling here should absolutely be automatic to make it easier for the player. Nucleus should also be underlined or bolded as it is the most important word in this whole text.